lib/api/element-commands/doubleClickElement.js

```diff
@@ -0,0 +1,56 @@
+const BaseElementCommand = require('./_baseElementCommand.js');
```

Comment:
I don't think we need this command. The "doubleClick()" should handle elements as well, if not rename this one to doubleClick(), add the functionality from the protocol command here.

Reply:
I renamed `doubleClickElement` to  `doubleClick` and in protocol added `elementIdDoubleClick`, does that work?